src/demos/confirm/confirm-demo.component.html

```diff
@@ -41,5 +41,5 @@
 </div>
 
 <sky-alert *ngIf="selectedAction">
-  You selected "{{ selectedAction }}".
+  You selected button "{{ selectedText }}" which has an action of {{ selectedAction }}" .
```

Comment:
You selected button "{{ selectedText }}" which action is "{{ selectedAction }}."

Can the period go inside the quotation marks because it technically goes inside and not outside them.